Docs/UserGuide/userguide.tex

```diff
@@ -474,7 +474,17 @@ \subsubsection*{Unix}
 \% make -j4
 }
 
-This will build both the QuantExt library and test suite.
+This will build both the QuantExt library and test suite using the
+QuantLib code you compiled in the previous step.  If you already have
+an installed version of QuantLib on your machine and you want to use
+it, you can change the {\tt ./configure} command above to
+
+{\tt\footnotesize
+\% ./configure --with-quantlib-prefix=/usr/local
+}
+
+where you can replace {\tt /usr/local} with the path where you
+installed QuantLib.
```

Comment:
This should probably also mention that if you're choosing an installation, it can't be a download or clone of the source; it has to have been made with `make install`, since the compiler is going to look into `PREFIX/include` and `PREFIX/lib`.  I wasn't sure how much to explain, though.  Suggestions are welcome.